addons/kafka/1.x/kafka-1.yaml

```diff
@@ -24,5 +26,5 @@ spec:
       enabled: true
   kudoReference:
     package: kafka
-    repo: https://kudo-repository.storage.googleapis.com/0.10.0
```

Comment:
We can't change this, it will break the revision (we need to make this a retroactive change)